### ISSUE TITLE
native syslog support

### ISSUE THREAD

**Issue by dch** [h2o/h2o#1124]
Either /dev/log or syslog over TCP (or UDP) would be super useful. Re-reading and processing the logfiles via rsyslog again is a little silly when h2o could send them directly.

**Comment by deweerdt** [comment]
I've used `access-log: "| logger -t h2o -d"` as a log directive when i needed to log directly to syslog.

**Comment by dch** [comment]
@deweerdt yes that is minimally functional.

native syslog would be better, for example this is what I get across multiple servers when using logger:

> 2016-11-21T09:28:27.943396+00:00 localhost h2o: 127.0.0.1 - - [21/Nov/2016:09:28:27 +0000] "GET / HTTP/2" 200 290 "-" "curl/7.51.0"

I can't tell which server this actually occurred on without log-rewriting, and again we are handling the logging content in 3 processes - h2o, logger, rsyslog.

**Comment by kazuho** [comment]
@dch Sorry for the belated response.

Would it be a viable option for you to specify a custom format using the `format` attribute? https://h2o.examp1e.net/configure/access_log_directives.html#access-log

**Comment by dch** [comment]
Yes I can use the custom format to get the server name into the logs, however what I'd like to avoid is that h2o writes a log file entry out to syslog, and then immediately in rsyslog, I need to rewrite that to incorporate the hostname to replace the `127.0.0.1` that comes in via logger or similar tools.

With custom access log formatting I can add tags, but I still later to use a regex in the logging system to extract the hostname from the tag, and then re-write it later on in the correct place. We're not a a webscale company so this is only really a tidiness feature for me rather than a performance one, but it seems unnecessary to rewrite the log data immediately to make it useful.
